src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  border: 1px #f3f3f3 solid;
+  border-radius: 5px;
+  width: 200px;
+  padding-inline: $card-padding;
+
+  &__image {
+    position: relative;
+    justify-content: center;
+    width: 160px;
+    height: 134px;
+    margin: 32px 3px 0;
+  }
+
+  &__info {
+    &-title {
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      margin-top: 40px;
+      letter-spacing: 0%;
+      margin-bottom: 0;
+    }
+
+    &-product-code {
+      margin-top: 4px;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: #616070;
+      letter-spacing: 0%;
+    }
+  }
+
+  &__review {
+    margin-top: 16px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    &-count {
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      letter-spacing: 0%;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+
+    &-name {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      letter-spacing: 0%;
+    }
+
+    &-number {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      letter-spacing: 0%;
+    }
+  }
+
+  &__button {
+    box-sizing: border-box;
+    height: 40px;
+    margin: 16px 0;

```

Comment:
instead bottom margin better to set bottom padding for.card. so card will have paddins for each side and keep content inside.

check all vertical margins for card elements. set them in one direction (for example all margin bottom)